operator/v1/0000_70_cluster-network-operator_01_crd.yaml

```diff
@@ -1,443 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
please don't rename files and change them in the same commit. I cannot review this easily now.